patches/series

```diff
@@ -4,3 +4,4 @@ fix-dpdk-mbuf-release.patch
 dpdk-config.patch
 multiop-includes.patch
 remove-java.patch
+tickets
```

Comment:
tickets doesn't seem like a meaningful name, and doesn't fit with the rest of the patches. `make-rc-test.path` or something similar seems better.

Reply:
Renaming to match convention of other patches.